### PR TITLE
move image pushing to script

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Move image pushing to script.
Additionally this will push to docker & quay.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: containers/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM docker:18.09.0-dind
+
+RUN apk add --no-cache -U ca-certificates wget unzip jq bash
+RUN wget https://releases.hashicorp.com/vault/0.11.5/vault_0.11.5_linux_amd64.zip
```
Comment (alvaroaleman) [inline containers/docker/Dockerfile]: Is the backslash key on your keyboard broken? :D

**Review by alvaroaleman — DISMISSED** [review]

File: containers/docker/Dockerfile (hunk 1 above)
Comment (mrIncompetent) [reply containers/docker/Dockerfile]: https://github.com/kubermatic/kubermatic/pull/2399/commits/51a568d595718070385d5554b455d27096b94d2b

Here you go :D

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**